scalafix-diff/shared/src/test/resources/git3.diff

```diff
@@ -0,0 +1,3181 @@
+diff --git a/project/Mima.scala b/project/Mima.scala

```

Comment:
I'm not sure this is a sustainable approach, can try to make these tests more targeted to the discovered bugs?